portality/static/js/doaj.fieldrender.edges.js

```diff
@@ -2195,9 +2195,15 @@ $.extend(true, doaj, {
                         this.currentQueryString = qs.queryString || "";
                     }
                 }
-                var frag = "<li class='alert'><p>You searched for <i>'";
+                var frag = "<li class='alert'><p>You searched for <i>";
                 frag += edges.escapeHtml(this.currentQueryString);
-                frag +="'</i> and we found no results.</p><p>Search terms must be in <strong>English</strong>.</p> <p>Try removing some of the filters you have set, modifying the text in the search box, or using less specific search terms.</p></li>";;
+                frag += "</i> and we found no results.</p>";
+                frag += "<p>Please check the following:</p><ul>";
```

Comment:
Check the following / check the spelling is a bit too repetetive for my tastes, but that's not a blocker!

Reply:
Change to

'Please *try* the following:' and then 
'Remove some of the filters you have set' and 
'Do your search again in English as...'

Reply:
Done!